src/FlipMove.js

```diff
@@ -397,49 +416,55 @@ class FlipMove extends Component {
 
       // If the placeholder was holding the container open while elements were
       // leaving, we we can now set its height to zero.
-      if (this.heightPlaceholderData.domNode !== null) {
-        this.heightPlaceholderData.domNode.style.height = 0;
+      if (this.heightPlaceholderData.domNode) {
+        this.heightPlaceholderData.domNode.style.height = '0';
       }
     }
   }
 
-  formatChildrenForHooks() {
-    const elements = [];
-    const domNodes = [];
+  callChildrenHook(hook: ChildrenHook) {
+    const elements: ElementShape[] = [];
+    const domNodes: (?HTMLElement)[] = [];
```

Comment:
Can we use the longform `Array<>` form when maybe types are involved? Feels cleaner to me.

(I was actually going to ask that we convert to it everywhere, as it's the format preferred by Facebook internally to avoid exactly this kind of ambiguity, but I didn't wanna add more work to your plate. If you're energetic enough, I'd appreciate it, but it's by no means necessary).